modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,79 @@
+= Share an Asset
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+:page-aliases: to-share-an-asset-with-a-user.adoc
+
+Exchange administrators and asset administrators can share an asset with another user in their organization, with the whole organization, or with an external organization.
+
+Any Exchange user can click the *Shared with me* link in the left navigation menu of Exchange to show all assets shared with their user account and all assets shared with their organization.
+
+== Share With a User
+
+If the person is not in your organization, click *Access Management* > *Users* to add them.
+
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of the person you want to share this asset with, and select that person in the list.
+. Select whether to grant the user Viewer, Contributor, or Admin rights.
+. Repeat steps 3 and 4 to share with additional users.
+. Click *Save*.
+
+You can also click *Share* to change who you are sharing the asset with or to stop sharing with one or more users.
+
+== Share With Your Organization
+
+Exchange administrators and asset administrators can share an asset to everyone in their organization.
+
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of your organization.
+. Select the list item for everyone at your organization.
+. Select whether to grant Viewer, Contributor, or Admin rights.
+. Click *Save*.
+
+When an asset is shared with an organization, email notification is sent to the organization administrators.
+
+You can also click *Share* to stop sharing the asset with everyone at the organization.
+
+== Share With an External Organization
+
+Exchange administrators and asset administrators can grant viewer access for an asset to everyone in a chosen external organization.
+
+The organization granting viewer access is the granting organization, and the organization that receives access is the receiving organization.
+
+. The receiving organization adds the granting organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
++
+Assets can only be shared with another organization if that organization has enabled receiving viewer access to assets from external organizations.
++
+. The granting organization adds the receiving organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
```

Comment:
DITTO prev. comment about quotation marks.
This statement follows the previous so closely, and cross-refers to the same location, that I wonder if we can say that each org. adds the other to their External Access lists?

Reply:
Much better! I merged these.